build.gradle

```diff
@@ -25,7 +25,6 @@ plugins {
 ext.ballerinaLangVersion = project.ballerinaLangVersion
 ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
 ext.testngVersion = project.testngVersion
-ext.slf4jVersion = project.slf4jVersion
```

Comment:
Shall we remove other ext.* definitions as those definitions are no need?

Reply:
Removed in 5f05f74